src/protobuf-net/Meta/MetaType.cs

```diff
@@ -2028,6 +2030,7 @@ public bool IsGroup
             else if (Type.IsEnum)
             {
                 var enums = GetEnumValues();
+                string enumNamePrefix = $"{GetSchemaTypeName(callstack)}_";

```

Comment:
suggestion: `= isEnumNamePrefixSupported ? $"{GetSchemaTypeName(callstack)}_" : "";` - that way the later uses can be simplified to just `.Append(enumNamePrefix).Append(member.Name)`, which simplifies the code and avoids a bunch of unnecessary string allocations